docs/competition/team-support/tinker-time.md

```diff
@@ -1,13 +1,23 @@
+Each 'slot' is usually 10 minutes, depending on how much time is scheduled and how many teams are expected. For each slot, team may book a single corner (without preference). Teams should be considerate of other teams and robots in the arena, using the space around the arena to watch rather than all trying to be in the arena at once. Shepherds will not be used to fetch teams for tinker time slot - they will need to remember.
 
-To book Tinker Time teams must find the Tinker Time Coordinator, stationed by a whiteboard near the staging area, who will allocate teams slots based on what's available.
+After their slot has elapsed, the team should vacate the arena (even if there isn't another team to replace them). Depending on the number of available slots, teams may be prevented from booking multiple slots.
 
-Each 'slot' will last 10 minutes, 4 teams can be in the arena at once (one in each corner), robots must enter the arena through staging. For SR2019, where we have a scaffold-based arena, during Tinker Time the second door will be opened and non-robot-carrying competitors may enter and exit through here.
+Tinker time can be booked at either [Helpdesk](./helpdesk.md) or Staging. Teams are free to cancel or postpone their slots as necessary, however they may risk losing their chance to test.
 
-Tinker Time starts directly after the welcome talk, however this means some teams will not be safety tested yet. If a team does not have a High Vis the Badgers in staging should contact the Health and Safety coordinator to see if there is a reason a team is missing it. If the reason is that the robot has yet to be tested, then the Badgers can make a judgement on the safety of the robot erring on the side of caution. If a team is rejected by a Badger they must get approval from the Health and Safety coordinator before entering the arena.
+Robots must be Safety checked (denoted by a high-vis) before they can enter the arena for Tinker Time, however do not necessarily need to pass the Regulations check. Teams which have not had a Safety check yet should be sent to helpdesk.
 
-On Saturday teams can book a limited number of slots directly after the welcome talk. On the Sunday teams can find the Tinker Time Coordinator and book a slot if one is available.
+Unless otherwise authorised by the Event Director, Tinker Time will not run during Lunch.
+
+## Test Arena
+
+The Test Arena is an arena which is open throughout the event for testing, rather than during scheduled times.
+
+The Test Arena 'slots' are usually 15 minutes - longer than regular Tinker Time. If, after 15 minutes, there are no teams waiting, the team may continue testing. If a new team turns up, the team who has been in the longest should be swapped out.
+
+Slots are booked at the arena, rather than helpdesk. Whilst there is no formal form for scheduling, however it's recommended to track the team's TLA and start time to easily see who is to be replaced. A whiteboard is best for this.
```

Comment:
```suggestion
Slots are booked at the arena, rather than helpdesk. Whilst there is no formal form for scheduling, it's recommended to track the team's TLA and start time to easily see who is to be replaced. A whiteboard is best for this.
```